### ISSUE TITLE
Show different Icon during checking for changes

### ISSUE THREAD

**Issue by GekoCH** [nextcloud/desktop#619]
while checking for changes don't show the green "everything is synch" icon pls show the blue synch Icon so we know that it is currently checking wehter it needs to sync or not

This Image is shown
![image](https://user-images.githubusercontent.com/3285638/44960125-e234c900-aef9-11e8-8dab-b49175eaac59.png)

during this task of Nextcloud
![image](https://user-images.githubusercontent.com/3285638/44960138-43f53300-aefa-11e8-8c5d-fbe80ba1f2d3.png)

**Comment by camilasan** [comment]
Hi! Which version of the client you are running?

**Comment by GekoCH** [comment]
the current version: 2.3.3

**Comment by GoetheG** [comment]
Never really thought about that but I saw that ob my computer too. I am using the current version too anf I can verify this behaviour.

By the way: Any news on the new version of the desktop client?

**Comment by camilasan** [comment]
> By the way: Any news on the new version of the desktop client?

Yes, we have a beta 2 atm and I think we will be able to release beginning of October.

**Comment by alexanderdd** [comment]
Can someone check with the latest client please?

**Comment by alexanderdd** [comment]
@er-vin this is not an enhancement, it is a bug, and one of the most severe bugs a sync client can have. The worst would be not syncing correctly, second worst is telling the user it is synced when actually it is not, which this report is about.

In my organisation people complain about this about once per month, and it leads to file conflicts (and frustrated users) about 4 times a month. Also, this report is **two** years old. 

I can confirm this report for multiple machines, Windows 10, client 2.6.5. 

Can someone please test this with latest release 3.0.2?

**Comment by archont00** [comment]
The same bug on Linux version 3.0.3-2 (Arch Linux).

**Comment by LukeOwlclaw** [comment]
This problem still persists with version 3.1.3.

![image](https://user-images.githubusercontent.com/2737900/113563490-14ee5300-9608-11eb-9277-d9d98bda15e0.png)

**Comment by timo-ecm2** [comment]
While my W10 laptop with the new Desktop client 3.2 checks for changes it shows in the System Tray "the Blue sync icon" and not "the Green Everything is synced icon".

user account Operating system: Windows 10
Operating system: Linux
Database: m=MariaDB
Nextcloud version: 20.0.8
Updated from an older Desktop Client or a fresh install

**Comment by archont00** [comment]
NC client 3.2.1 Archlinux. The bug still is the same.

**Comment by alexanderdd** [comment]
please reopen @er-vin or @FlexW ?

**Comment by timo-ecm2** [comment]
Great!

**Comment by trvrcr** [comment]
I've tried to recreate the bug in v3.2.2 (windows) but wasn't able to:
- Added folder to sync structure
- Removing folder to sync structure
- Modify file and save it
There is a latency of a second or 2 before the tray icon changes to syncing, but that is imho acceptable.

When the modified files are small and thus the syncing is fast, the icon in the tray doesn't have the time to change.
And when hovering you only get the message "Nextcloud: Last Sync was successful."

It might be better to also give a time in this message

e.g. "Nextcloud: Last Sync (2 sec ago) was successful."
"Nextcloud: Last Sync (20 min ago) was successful."

**Comment by LukeOwlclaw** [comment]
Today I this very problem occurred again with version 3.4.1:

![image](https://user-images.githubusercontent.com/2737900/148892320-828c99b1-3e33-4dc4-a9a2-023813be28b6.png)

**Comment by GekoCH** [comment]
same here, is this really a bug or a "feature" that we don't know the reason for?
It shouldn't ne that hard to fix this...

**Comment by claell** [comment]
Hm, just to clarify: Is this only happening with multiple accounts used or also the case for single accounts?

**Comment by GekoCH** [comment]
nope I just have one account connected

**Comment by claell** [comment]
Alright, thanks!

**Comment by Rello** [comment]
Closing as completed after an automated backlog review. The current tray-status implementation maps the initial and sync-preparation states to the active syncing state, so checking for changes is represented by the syncing icon rather than the green completed icon.
